Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs

```diff
@@ -261,18 +277,24 @@ public static IServiceCollection AddJellyfinApiSwagger(this IServiceCollection s
         {
             return serviceCollection.AddSwaggerGen(c =>
             {
-                c.SwaggerDoc("api-docs", new OpenApiInfo
+                var provider = serviceCollection.BuildServiceProvider().GetRequiredService<IApiVersionDescriptionProvider>();
+
+                // Add a swagger document for each discovered API version
+                foreach (var description in provider.ApiVersionDescriptions)
                 {
-                    Title = "Jellyfin API",
-                    Version = "v1",
-                    Extensions = new Dictionary<string, IOpenApiExtension>
+                    c.SwaggerDoc($"{description.GroupName}", new OpenApiInfo()
```

Comment:
```suggestion
                    c.SwaggerDoc(description.GroupName", new OpenApiInfo()
```

Reply:
isn't there a obsolete `"` in your code suggestion?

Reply:
Thanks, fixed